app/assets/stylesheets/site/globals/site.variables

```diff
@@ -1,3 +1,12 @@
 /*******************************
      User Global Variables
 *******************************/
+
+@ows_pink: #D60B52;
+@ows_purple: #3E2153;
+@ows_blue: #29235C;
+
+@ows_primaryColor: @ows_pink;
+
+@primaryColor: @ows_primaryColor;
+
```

Comment:
I would suggest having a consistent naming scheme here: `snake_case` or `camelCase` but not both. My preference goes to `snake_case`:

`@ows_primary_color`